Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs

```diff
@@ -81,13 +81,43 @@ public static void DoBuildUniversal (string outputPath)
             {
                 throw new Exception(error); // ensures exit code != 0.
             }
+#if UNITY_EDITOR
+            //Unity creates unused assets no matter what
+            //this hack removes those assets after the compilation
+            RemoveIncorrectAssets(outputPath);
+#endif
         }
+        static void RemoveIncorrectAssets(string outputPath)
+        {
+            string projectName = PlayerSettings.productName;
+            string assetsPath = outputPath + "/" + projectName + "/" + projectName + ".Windows/Assets/";
+
+            //removing unused assets for windows
+            if (File.Exists(assetsPath + "WideTile.scale-100.png"))
+                File.Delete(assetsPath + "WideTile.scale-100.png");
+
+            if (File.Exists(assetsPath + "MediumTile.scale-100.png"))
+                File.Delete(assetsPath + "MediumTile.scale-100.png");
+
+            //removing unused assets for windowsPhone
+            assetsPath = outputPath + "/" + projectName + "/" + projectName + ".WindowsPhone/Assets/";
```

Comment:
Please check with Freddy or Keith but I think the Windows folders locations in relation to the Unity Asset folder may vary from project to project (there's a script that's is used to set up a new project, and you define these paths when running it), so it won't work for every project.